src/config.js

```diff
@@ -15,8 +15,10 @@ const {
 function getCoveragePaths() {
     let paths = [];
     const modulePaths = getModulePaths();
-    modulePaths.forEach((p) => {
-        paths.push(path.join(p, '**/*.js'));
+    modulePaths.forEach(p => {
+        // convert back to forward slashes here on Windows for Jest  to be happy
+        p = p.replace(/\\/g, '/');
```

Comment:
why not using `path.normalize()` instead?

Reply:
It only works with forward slashes, even on Windows. From my understanding, `normalize` won't replace backslashes with forward slashes.